### ISSUE TITLE
Action on tool delete

### ISSUE THREAD

**Issue by hansioan** [bio-tools/content#203]
@bgruening @hmenager @joncison @matuskalas @piotrgithub1 

Hi all,

This issue is regarding tool deletion from the ecosystem, especially pull requests coming from outside (e.g. from bio.tools or from forks).

It could be the case that in our content file structure we have something like:
`data/tool_id/tool_id.biotools.json` where this is the only file. If we delete this file then the whole folder goes as well, if this is intended behaviour it's fine but we just have to agree on what happens on delete. 

I was thinking of, istead of deleting the file immediately, especially when changes are coming from bio.tools perhaps we can mark tools as deleted by renaming the bio.tools files to something like `.deleted.tool_id.biotools.json`. In this way the folder doesn't go away.

I think we can code and handle both cases, it's just that we have to agree on what we want the functionality to be. What are your opinions on this? Please tag other people.

**Comment by hansioan** [comment]
I agree with @piotrgithub1 suggestion that we rename the entry rather than delete it, but keep the `.deleted` part at the end, i.e. `tool_id.biotools.json.deleted`
In this way the file won't have a `.json` extension, but a `.deleted` extension which will separate it from regular `.json` files.

**Comment by mihai-sysbio** [comment]
It might also depend with the root cause of the deletion.
If it's a mistake such as a duplicated entry, I'd remove it completely.
If the entry is inaccessible, I would be tempted to create another field to mark it as inactive. This is based on the assumption that any project may become inactive at a certain point, but that doesn't mean it should be automatically ignored by having the extension changed.
Moreover, the `.deleted` approach appears rather binary. We might find that there is a gradient there with regards to the possible states.
